cdatransform/transform/pdclib.py

```diff
@@ -137,11 +141,12 @@ def harmonized_files(files, case):
         this_file["identifier"] = [{"value": this_file.get("file_id"), "system": "PDC"}]
         this_file["drs_uri"] = "".join(["drs://dg.4DFC:",this_file.get("file_id")])
         this_file["id"] = this_file.pop("file_id")
-        this_file["associated_project"] = (case.get("project_submitter_id"),)
+        this_file["associated_project"] = case.get("project_submitter_id")
```

Comment:
Is this correct? The corresponding code in gdclib creates an array here. 
```
        this_file["associated_project"] = [case.get("project", {}).get("project_id")]
```

Reply:
I think the associated project field is a bit in limbo. At some point we will want to look at the specimen info and may find that the specimen is associated with more than one project, hence a list, but that may not be the case the vast majority of the time